### PR TITLE
UCP/MD: fix coding style

### PR DESCRIPTION
**Pull request by evgeny-leksikov**
## What
fix coding style, basically too long lines

### CODE REVIEW

File: src/ucp/wireup/select.c [hunk 1]
```diff
@@ -1475,7 +1475,8 @@ ucs_status_t ucp_wireup_select_sockaddr_transport(ucp_ep_h ep,
 
         md_index = resource->md_index;
         md       = context->tl_mds[md_index].md;
-        ucs_assert(context->tl_mds[md_index].attr.cap.flags & UCT_MD_FLAG_SOCKADDR);
+        ucs_assert(context->tl_mds[md_index].attr.cap.flags &
+                   UCT_MD_FLAG_SOCKADDR);
 
         if (uct_md_is_sockaddr_accessible(md, &params->sockaddr, UCT_SOCKADDR_ACC_REMOTE)) {
```
Comment (dmitrygladkov) [inline src/ucp/wireup/select.c]:
```
if (uct_md_is_sockaddr_accessible(md, &params->sockaddr,
                                                       UCT_SOCKADDR_ACC_REMOTE)) {
```
?

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/9136/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/9137/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/9138/ for details (Mellanox internal link).

**Pull request merged**